test/common/runtime/uuid_util_test.cc

```diff
@@ -35,18 +35,23 @@ TEST(UUIDUtilsTest, mod) {
   EXPECT_EQ(7295, result);
 }
 
+TEST(UUIDUtilsTest, benchmark) {
```

Comment:
My point mainly was that this "test" doesn't actually test anything. There are no assertions, etc. It will just burn CPU and always pass. (And from a perf perspective it will always return a different result depending on where it is run). So, typically for this type of thing I would make it a DISABLED test so that it can be run for local testing, but doesn't waste time in general.